.github/workflows/cli-test.yml

```diff
@@ -0,0 +1,108 @@
+name: Run CLI tests
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+  push:
```

Comment:
Should this be `pull_request` instead? Or additionally?

Reply:
🤦‍♀️ It should be pull_request, like in the main workflow.